modules/adding-an-odh-instance.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='adding-an-odh-instance_{context}']
+= Adding an Open Data Hub instance 
+
+[role='_abstract']
+Add an Open Data Hub instance to your project to be able to give access to an Open Data Hub dashboard.
+
+.Prerequisites
+* You have installed the Open Data Hub Operator.
+* You have created a new project for your instance of Open Data Hub.
+
+.Procedure
+. In the OpenShift web console, select *Operators* -> *Installed Operators*.
+. Click *Project* and select the *odh* project.
+. Find and click the *Open Data Hub Operator* to display the details for the currently installed version.
+. In the *KfDef* tile, click *Create instance*. `KfDef` is a specification designed to control the provisioning and management of Kubeflow deployment. The default `KfDef` configuration deploys the latest release of the Open Data Hub core components. For more information, see link:https://opendatahub.io/docs/tiered-components[Tiered Components].
```

Comment:
Is this accurate? 
"KfDef is a specification designed to control the provisioning and management of Kubeflow deployment. The default KfDef configuration deploys the latest release of the Open Data Hub core components."

Reply:
I think so, but it might be worth getting an SME review of this statement in the ODH slack workspace.

Apart from the technical validity, I'd make a couple of minor tweaks:

" A `KfDef` file is a specification designed to control provisioning and management of a Kubeflow deployment.   A `KfDef` file is created when you install Open Data Hub Operator. This default configuration deploys the latest release of the Open Data Hub core components."